doc/api/stream.md

```diff
@@ -1273,8 +1273,8 @@ If the `decodeStrings` property is set in the constructor options, then
 indicate the character encoding of the string. This is to support
 implementations that have an optimized handling for certain string
 data encodings. If the `decodeStrings` property is explicitly set to `false`,
-the `encoding` argument can be safely ignored, and `chunk` will always be a
-`Buffer`.
+the `encoding` argument can be safely ignored, and `chunk` will remain the same
+type from the source.
```

Comment:
How can one tell what the type is? I thought these were always Buffers lol.

Do you mean it is different for Object Mode?